### PR TITLE
feat(http): allow signal path as onProgress

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: packages/node_modules/@cerebral/http/README.md [hunk 1]
```diff
@@ -338,7 +338,7 @@ function someDeleteAction ({http, props}) {
     name: 'filename.png', // Default to "files"
     data: {}, // Additional form data
     headers: {},
-    onProgress(progress) {} // Upload progress
+    onProgress: 'some.signal.path' // Upload progress
```
Comment (Guria) [inline packages/node_modules/@cerebral/http/README.md]: shouldn't we have both variants in docs?

File: packages/node_modules/@cerebral/http/README.md [hunk 2]
```diff
@@ -355,12 +355,13 @@ export default [
...
   httpUploadFile('/uploads', props`file`, {
-    name: state`currentFileName`
+    name: state`currentFileName`,
+    onProgress: 'some.signal.path'
   }), {
     success: [],
     error: [],
     abort: [], // Optional
-    '${STATUS_CODE}': [] // Optionally any status code, ex. 404: []
+    $STATUS_CODE: [] // Optionally any status code, ex. 404: []
```
Comment (Guria) [inline packages/node_modules/@cerebral/http/README.md]: didn't get this change. looks strange

Comment (Guria) [inline packages/node_modules/@cerebral/http/README.md]: both sides strange :)

File: packages/node_modules/@cerebral/http/README.md (hunk 1 above)
Comment (christianalfoni) [inline packages/node_modules/@cerebral/http/README.md]: There is really no reason to trigger anything else than a signal :) This was method thing was just some artefact from previous implementations

File: packages/node_modules/@cerebral/http/README.md (hunk 2 above)
Comment (christianalfoni) [inline packages/node_modules/@cerebral/http/README.md]: It indicates a placeholder :) No idea how to express it better?

```js
{
  success: [],
  error: [],
  [STATUS_CODE]: [] // ??
}
```

Comment (Guria) [inline packages/node_modules/@cerebral/http/README.md]: it must be valid js. and your last suggestion looks much better now

Comment (christianalfoni) [inline packages/node_modules/@cerebral/http/README.md]: Cool, let me update :)

**Comment by christianalfoni** [comment]
Arright, updated :)

**Pull request merged**